src/pages/getting_started.tsx

```diff
@@ -1,46 +1,55 @@
-import { UserLayout } from '@layout';
-import { NextPage } from 'next';
-import { Card } from 'react-bootstrap';
+import { UserLayout } from "@layout";
+import { NextPage } from "next";
+import { Card } from "react-bootstrap";
 
 const GettingStarted: NextPage = () => (
   <UserLayout>
+    <Card.Title>Getting Started</Card.Title>
+    <Card.Text>
+      Follow the instructions below to install the Martian environment, upload
+      your algorithm submissions, request scrimmages with other players, and
+      check out your match results - happy hacking!
+    </Card.Text>
     <Card className="mb-3">
       <Card.Body>
-        <Card.Title>Getting Started</Card.Title>
+        <Card.Title>Installation</Card.Title>
         <Card.Text>
-          Follow through the instructions below to learn more about installation
-          instructions, how you can upload your bot submissions, request
-          scrimmages with other players, and check out your match results!
+          Visit our AWAP 2023{" "}
+          <a href="https://github.com/ACM-CMU/awap-viewer-2023">Viewer</a> and{" "}
+          <a href="https://github.com/ACM-CMU/awap-engine-2023">Engine</a>{" "}
+          GitHub repos to install the Martian environment. You'll need Python
```

Comment:
Maybe we should link the game guide here as well.

Reply:
added!